template/cordova/lib/run.js

```diff
@@ -39,7 +39,7 @@ module.exports.run = function (options) {
 
     // parse arg
     var args  = nopt({
-        'archs': String,
+        'archs': [String],
```

Comment:
Why do we need to change this to Array?
What will the command usage become with this change (could you please give an example)?
Also does it make sense to also update the tests and the docs if we make this change?